tensorflow/lite/micro/kernels/strided_slice_test.cc

```diff
@@ -289,7 +289,7 @@ TF_LITE_MICRO_TEST(In1D_BeginMask) {
   float golden[] = {1, 2, 3};
   float output_data[4];
 
-  TfLiteStridedSliceParams builtin_data = {1, 0, 0, 0, 0};
+  TfLiteStridedSliceParams builtin_data = {1, 0, 0, 0, 0, 0};
```

Comment:
Should these be `false` instead of `0` since the added field `offset` is a bool?

Reply:
yes, I completely missed that it was a bool. will fix in a bit.